### PR TITLE
Several updates

### PR DESCRIPTION
**Pull request by ymesika**
Following a performance session I had to fix several issues.
Also following a thread that mentioned that iperf isn't a good choice for TCP latency tests I separated the TCP latency from throughput and now using fortio for latency and iperf3 for the throughput.

### CODE REVIEW

**Comment by kdorosh** [comment]
cc @GregHanson

**Review by dhawton — APPROVED** [review]
LGTM

File: lib/tests.sh [hunk 1]
```diff
@@ -211,7 +164,7 @@ EOF
 
     sleep 10
 
-    kctl -n $TESTING_NAMESPACE wait pods -l ambient-type=waypoint --for condition=Ready --timeout=120s
+    kctl -n $TESTING_NAMESPACE wait pods -l gateway.istio.io/managed=istio.io-mesh-controller --for condition=Ready --timeout=120s
```
Comment (GregHanson) [inline lib/tests.sh:167]: not sure if we want to do away with the Gateway yaml and just use istioctl:
https://github.com/solo-io/ambient-performance/commit/157f3cee143dd7cd347da24872fabcaef8a1160f#diff-0a8547e24c9c9a12f94240309758426ee0dfcd5e7ce9d9f8de04e86ae60b5682
```
runIstioctl x waypoint apply -n $TESTING_NAMESPACE --service-account=benchmark-server
```

File: lib/tests.sh [hunk 2]
```diff
@@ -207,11 +156,15 @@ EOF
     fi
 
     log "Applying Waypoint Proxy"
-    kctl apply -n $TESTING_NAMESPACE -f "$wp"
+    if [[ ! -z "$1" ]]; then
```
Comment (GregHanson) [inline lib/tests.sh:159]: lgtm

**Review by GregHanson — APPROVED** [review]

**Pull request merged**